docs/_build/html/installation.html

```diff
@@ -75,97 +95,107 @@
       <div class="wy-nav-content">
+        
         <div class="rst-content">
-          <div role="navigation" aria-label="breadcrumbs navigation">
+        
+          
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+<div role="navigation" aria-label="breadcrumbs navigation">
+
   <ul class="wy-breadcrumbs">
-    <li><a href="index.html">Docs</a> &raquo;</li>
-      
-    <li>Installation</li>
+    
+      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
+        
+      <li>Installation</li>
+    
+    
       <li class="wy-breadcrumbs-aside">
         
-          <a href="_sources/installation.txt" rel="nofollow"> View page source</a>
+            
+            <a href="_sources/installation.rst.txt" rel="nofollow"> View page source</a>
+          
         
       </li>
+    
   </ul>
+
+  
   <hr/>
 </div>
-          <div role="main" class="document">
+          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
+           <div itemprop="articleBody">
             
   <div class="section" id="installation">
 <h1>Installation<a class="headerlink" href="#installation" title="Permalink to this headline">¶</a></h1>
 <div class="section" id="installing-via-pip">
 <h2>Installing via pip<a class="headerlink" href="#installing-via-pip" title="Permalink to this headline">¶</a></h2>
 <p>To install the PySD package from the Python package index into an established
 Python environment, use the pip command:</p>
-<div class="highlight-bash"><div class="highlight"><pre><span></span>pip install pysd
+<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>pip install pysd
 </pre></div>
 </div>
 </div>
 <div class="section" id="installing-from-source">
 <h2>Installing from source<a class="headerlink" href="#installing-from-source" title="Permalink to this headline">¶</a></h2>
 <p>To install from the source, clone the project with git:</p>
-<div class="highlight-bash"><div class="highlight"><pre><span></span>git clone https://github.com/JamesPHoughton/pysd.git
+<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>git clone https://github.com/JamesPHoughton/pysd.git
 </pre></div>
 </div>
 <p>Or download the latest version from the project webpage: <a class="reference external" href="https://github.com/JamesPHoughton/pysd">https://github.com/JamesPHoughton/pysd</a></p>
 <p>In the source directory use the command</p>
-<div class="highlight-bash"><div class="highlight"><pre><span></span>python setup.py install
+<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span>python setup.py install
 </pre></div>
 </div>
 </div>
 <div class="section" id="required-dependencies">
 <h2>Required Dependencies<a class="headerlink" href="#required-dependencies" title="Permalink to this headline">¶</a></h2>
-<p>PySD is built on python 2.7, and may not work as advertized on 3.x.</p>
+<p>PySD was originally built on python 2.7, but will work with 3.x.</p>
```

Comment:
Can you create an issue to track documentation changes that will need to happen when we go to 1.0.0 and drop Py2, and attach it to the 1.0.0 milestone.

Reply:
Sure, I have detected a lot of things and maybe I will start changing them in a branch